src/content/guides/production.md

```diff
@@ -234,6 +192,37 @@ __src/index.js__
 ```
 
 
+## Minification
```

Comment:
Why this structure change? What did you changed exactly?

Reply:
for minification specifically or?
If yes:
-Add: webpack v4+ will minify your code by default in [`production mode`](/concepts/mode/#mode-production).
-Change: If you decide to try another minification plugin, just make sure your new choice also drops dead code as described in the [tree shaking](/guides/tree-shaking) guide and provide it as the [`optimization.minimizer`](/configuration/optimization/#optimization-minimizer).

Reply:
I mean, moving those 2 sections from the top to the bottom.